src/components/pirep-list-item/pirep-list-item.html

```diff
@@ -0,0 +1,79 @@
+<ion-item *ngIf="pirep">
+	<ion-row>
+		<ion-col>
+			<h1>
+				{{ pirep.aircraft }}
+				<span class="observation-time" *ngIf="pirep?.observationTime != ''">
+					({{ pirep?.observationTime.local().format('h:mm a') }})
+				</span>
+			</h1>
+		</ion-col>
+	</ion-row>
+	<ion-row *ngIf="pirep.altitude">
+		<ion-col>
+			<strong>Altitude (MSL):</strong> {{ pirep.altitude }}
+		</ion-col>
+	</ion-row>
+	<ion-row *ngIf="pirep.latitude && pirep.longitude">
+		<ion-col>
+			<strong>Location:</strong> Lat {{ pirep.latitude }} &nbsp; Long {{ pirep.longitude }}
+		</ion-col>
+	</ion-row>
+	<ion-row *ngIf="pirep.icingCondition.length > 0">
+		<ion-col col-12>
+			<h2>Icing</h2>
+			<ion-row *ngFor="let icing of pirep.icingCondition; let i = index">
+				<ion-col>
+					<div class="icing-intensity" *ngIf="icing.intensity || icing.type">
+						{{ icing.intensity }}
+						{{ icing.type }}
+					</div>
+					<div class="icing-base" *ngIf="icing.base || icing.top">
+						<span *ngIf="icing.base">{{ icing.base }} (Base)</span>
+						<span *ngIf="icing.base && icing.top">to</span>
+						<span *ngIf="icing.top">{{ icing.top }} (Top)</span>
+					</div>
+				</ion-col>
+			</ion-row>
+		</ion-col>
+	</ion-row>
+	<ion-row *ngIf="pirep.turbulenceCondition.length > 0">
+		<ion-col col-12>
+			<h2>Turbulence</h2>
+			<ion-row *ngFor="let turbulence of pirep.turbulenceCondition; let i = index">
```

Comment:
Not using i?

Reply:
Good catch. Left over from copied code. I will fix.